instances/service/CollaboratoryWiki.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/service/CollaboratoryWiki",
```

Comment:
Should reflect the name of the instance: "/EBRAINSCollaboratoryWiki"

Reply:
fixed